From ed5f1899bf1f3ea106e055695add2596e84bc0bc Mon Sep 17 00:00:00 2001
From: Sebastian Panceac <sebastian@balena.io>
Date: Thu, 4 Jul 2019 10:52:50 +0200
Subject: [PATCH] u-boot: Enable CMD_PART config

Upstream-status: Inappropriate[configuration]
Signed-off-by: Sebastian Panceac <sebastian@balena.io>
---
 configs/mx8mq_phanbell_defconfig | 1 +
 1 file changed, 1 insertion(+)

diff --git a/configs/mx8mq_phanbell_defconfig b/configs/mx8mq_phanbell_defconfig
index 74a6fdcc14..792cfe5b2a 100644
--- a/configs/mx8mq_phanbell_defconfig
+++ b/configs/mx8mq_phanbell_defconfig
@@ -42,3 +42,4 @@ CONFIG_ANDROID_BOOT_IMAGE=y
 CONFIG_ZIRCON_BOOT_IMAGE=y
 CONFIG_SD_BOOT=y
 CONFIG_CMD_IMPORTENV=y
+CONFIG_CMD_PART=y
-- 
2.17.1

